applets/wncklet/window-list.c

```diff
@@ -42,6 +42,9 @@ typedef struct {
 
 	GtkOrientation orientation;
 	int size;
+#ifndef WNCK_INPROCESS
```

Comment:
Typo our you have both `WNCK_INPROCESS` and `WNCKLET_INPROCESS`?

Reply:
duh, typo of course - good catch :)
I just fixed and pushed.